docs/09.EXT-REFERENCE-ARG.md

```diff
@@ -98,6 +98,69 @@ Enum that indicates whether an argument is optional or required.
 - [jerryx_arg_function](#jerryx_arg_function)
 - [jerryx_arg_native_pointer](#jerryx_arg_native_pointer)
 
+## jerryx_arg_round_t
+
+Enum that indicates the rounding policy which will be chosen to transform an integer.
+
+ - JERRYX_ARG_ROUND - use round() method.
+ - JERRYX_ARG_FLOOR - use floor() method.
+ - JERRYX_ARG_CEIL - use ceil() method.
+
+**See also**
+
+- [jerryx_arg_uint8](#jerryx_arg_uint8)
+- [jerryx_arg_uint16](#jerryx_arg_uint16)
+- [jerryx_arg_uint32](#jerryx_arg_uint32)
+- [jerryx_arg_int8](#jerryx_arg_int8)
+- [jerryx_arg_int16](#jerryx_arg_int16)
+- [jerryx_arg_int32](#jerryx_arg_int32)
+
+
+## jerryx_arg_clamp_t
+
+Enum that indicates whether it will throw error when the integer's range is
+out of the target type.
+
+ - JERRYX_ARG_CLAMP - it will try to clamp the number into the range.
+ - JERRYX_ARG_NO_CLAMP - it will throw range error.
```

Comment:
(doc) the descriptions are not word by word identical to the comments in the sources. it might be easier to maintain documentation if they would always be kept in sync. (btw, why is "try to clamp"? can clamping not succeed?)